src/antithesis_sdk/lifecycle.py

```diff
@@ -7,7 +7,7 @@
 from typing import Mapping, Any
 import json
 import sys
-from ._internal import dispatch_output
+from ._internal import dispatch_output, ANTITHESIS_SDK_VERSION
```

Comment:
Why do we need this here?

Reply:
We don't - good catch, and `pylint` also advised the same.  It has been removed.